src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Roboto-Regular.ttf");
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  font-style: medium;
+  src: url("../fonts/Roboto-Medium.ttf");
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 700;
+  font-style: bold;
```

Comment:
Change font-style value to 'normal' and remove 'bold' as it's not a valid font-style value. Use 'font-weight' for setting the font weight.